src/xbot/common/controls/sensors/XboxControllerWpiAdapter.java

```diff
@@ -65,7 +69,46 @@ public double getRightStickY(){
        
        return internalXboxController.getRawAxis(4);
    }
+
+   public boolean getLeftStickXInversion()
```

Comment:
I might just be missing it, but shouldn't these be used in all of the get calls (like `getLeftStickY()`) to actually apply the inversion?